src/backend/tests/projects.test.ts

```diff
@@ -150,4 +151,51 @@ describe('Projects', () => {
     expect(prisma.project.findMany).toHaveBeenCalledTimes(1);
     expect(res.body).toStrictEqual([]);
   });
+
+  test('setProjectTeam fails given invalid project wbs number', async () => {
+    const res = await request(app).post('/1.0.1/set-team').send({ submitterId: 1, teamId: 'test' });
+    expect(res.statusCode).toBe(400);
+    expect(res.body).toStrictEqual({ message: `1.0.1 is not a valid project WBS #!` });
+  });
+
+  test('setProjectTeam fails with invalid submitterId', async () => {
+    const res = await request(app).post('/1.2.0/set-team').send({ submitterId: -1, teamId: 'test' });
+
+    expect(res.statusCode).toBe(400);
+  });
+
+  test('setProjectTeam fails with invalid team id', async () => {
+    jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(null);
+
+    const res = await request(app).post('/1.2.0/set-team').send({ submitterId: 1, teamId: 123 });
+
+    expect(res.statusCode).toBe(400);
+  });
+
+  test('setProjectTeam fails with team that does not exist', async () => {
+    jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(null);
+    jest.spyOn(prisma.project, 'findFirst').mockResolvedValue({ ...project1 });
+
+    const res = await request(app).post('/1.2.0/set-team').send({ submitterId: 1, teamId: 'test' });
+
+    expect(res.statusCode).toBe(404);
+  });
+
+  test('setProjectTeam fails with no permission from submitter (guest)', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue({ ...wonderwoman, googleAuthId: 'b' });
```

Comment:
for all these users, you dont need to add the google auth id since they already all have it